voxel51/users/query.py

```diff
@@ -115,6 +115,20 @@ def add_search_or(self, field, search_strs):
         '''
         return self.add_search(field, "|".join(search_strs))
 
+    def add_generic_search(self, search_str):
```

Comment:
"generic" search is not very meaningful. Something can be generic in many ways. As a user scrolling through the docs, I am more likely to understand what this does if I see:

```py
    def add_search_any_field(self, search_str):
        '''Adds the given search that matches any field to the query.

        The resulting query will return results where any field matches the
        specified search string.

        ...
    ```
```